ui/components/app/snaps/snap-ui-markdown/snap-ui-markdown.js

```diff
@@ -23,7 +23,8 @@ const Paragraph = (props) => (
     variant={TextVariant.bodyMd}
     className="snap-ui-markdown__text"
     overflowWrap={OverflowWrap.Anywhere}
-    color={TextColor.inherit}
+    color={TextColor.textDefault}
+    textVariant={TextVariant.bodyMd}
```

Comment:
```suggestion
```

This is already applied via `variant` AFAIK

Reply:
You are right. I removed this one